crates/pevm/src/lib.rs

```diff
@@ -4,14 +4,49 @@
 
 // TODO: Better types & API for third-party integration
 
-use std::hash::{BuildHasher, BuildHasherDefault, Hash, Hasher};
+use std::{
+    future::Future,
+    hash::{BuildHasher, BuildHasherDefault, Hash, Hasher},
+    sync::OnceLock,
+    thread,
+};
+use tokio::runtime::{Handle, Runtime};
 
 use alloy_primitives::{Address, B256, U256};
 use bitflags::bitflags;
 use hashbrown::HashMap;
 use rustc_hash::FxBuildHasher;
 use smallvec::SmallVec;
 
+static GLOBAL_RUNTIME: OnceLock<Runtime> = OnceLock::new();
```

Comment:
I'm not a fan of global variables and `pevm` should not export something like `block_on`, so let's just promote the current `runtime` in `RpcStorage` from `Runtime` to `OnceLock<Runtime>`.